.env

```diff
@@ -1,4 +1,10 @@
+# Server
 API_HOST="http://localhost:5000"
 API_SERVER_PORT=5000
-CLIENT_PORT=3000
 
+# Auth 
+API_AUTH_HOST="http://localhost:5000"
```

Comment:
Not exactly sure what this is used for, but should this be 5050 here as well?

Reply:
I totally missed that ..